metagov/metagov/core/models.py

```diff
@@ -394,6 +409,11 @@ def update(self):
         - Call ``self.save()`` to persist changes."""
         pass
 
+    @property
```

Comment:
what is this being used for?

Reply:
It's being used by PolicyKit [here](https://github.com/mashton/policykit/blob/234848148d6414e6d5ac755b0eef979a98c1a602/policykit/policyengine/models.py#L543) to get the proxy model of a given GovernanceProcess. PK has a model with a ForeignKey relation to GovernanceProcess –– but in order to call the proxy functions that were defined by the plugin author, you need to get the proxy instance. If the linked code simply called `self.governance_process.close()`, that would just hit [this function](https://github.com/metagov/gateway/blob/f6a0ea887e13a63bd28b66f425affe272a1c34ac/metagov/metagov/core/models.py#L355).

I'd love a cleaner way to do this. I guess another way for PolicyKit to do it would be to do a plugin lookup and then call `plugin.get_process(id=self.governance_process.pk).close()` or `plugin.close_process(id=self.governance_process.pk)` but it just seemed more straightforward to me to expose an easy way to access the proxy.